src/commons/controlBar/ControlBarGitHubButtons.tsx

```diff
@@ -0,0 +1,102 @@
+import { ButtonGroup, Classes, Intent, Popover } from '@blueprintjs/core';
+import { IconNames } from '@blueprintjs/icons';
+import { Octokit } from '@octokit/rest';
+import * as React from 'react';
+
+import { GitHubFile, GitHubState } from '../../features/github/GitHubTypes';
+import { store } from '../../pages/createStore';
+import controlButton from '../ControlButton';
+
+export type ControlBarGitHubButtonsProps = {
+  loggedInAs?: Octokit;
+  currentFile?: GitHubFile;
+  isDirty?: boolean;
+  onClickOpen?: () => any;
+  onClickSave?: () => any;
+  onClickSaveAs?: () => any;
+  onClickLogIn?: () => any;
+  onClickLogOut?: () => any;
+  onPopoverOpening?: () => any;
+};
+
+const stateToIntent: { [state in GitHubState]: Intent } = {
+  LOGGED_OUT: Intent.NONE,
+  LOGGED_IN: Intent.NONE
+};
+
+export const ControlBarGitHubButtons: React.FC<ControlBarGitHubButtonsProps> = props => {
+  // The 'loggedInAs' is not used directly in this code block
+  // However, keeping it in will ensure that the component re-renders immediately
+  // Or else, the re-render has to be triggered by something else
+
+  const isLoggedIn = store.getState().session.githubOctokitInstance !== undefined;
+  const shouldDisableButtons = !isLoggedIn;
+  //const shouldDisableButtons = false;
```

Comment:
Commented out lines can be removed (there are some more underneath)

Reply:
Thanks for pointing this out.
Unnecessary comments are removed in 923ec68.